torchgeo/transforms/indices.py

```diff
@@ -22,119 +22,63 @@
 _EPSILON = 1e-10
 
 
-def ndbi(swir: Tensor, nir: Tensor) -> Tensor:
-    """Compute Normalized Different Built-up Index (NDBI).
+class AppendNormalizedDifferenceIndex(Module):
+    """Append normalized difference index as channel to image tensor."""
```

Comment:
Needs `.. versionadded:: 0.2` (or more likely 0.3)

Reply:
do these go in the class docstring or the constructor docstring?

Reply:
I would put them in the class docstring since the whole class is new. Depending on how you configure Sphinx these are equivalent though.